org.eclipse.xtend.lib.gwt/about.html

```diff
@@ -13,8 +13,8 @@ <h3>License</h3>
 
 <p>The Eclipse Foundation makes available all content in this plug-in (&quot;Content&quot;).  Unless otherwise 
 indicated below, the Content is provided to you under the terms and conditions of the
-Eclipse Public License Version 1.0 (&quot;EPL&quot;).  A copy of the EPL is available 
-at <a href="http://www.eclipse.org/legal/epl-v10.html">http://www.eclipse.org/legal/epl-v10.html</a>.
+Eclipse Public License Version 2.0 (&quot;EPL&quot;).  A copy of the EPL is available 
```

Comment:
@svenefftinge i also see a newer date in 
the about.htmls in other projects (e.g. emf/jdt/ltk)
so maybe we should simly copy the whole files over.